src/resources/Products/Product.ts

```diff
@@ -6,7 +6,10 @@ import {ProductsRequestModel, ProductsResultModel} from './ProductInterfaces';
 export default class Products extends Resource {
     static baseUrl = `/rest/organizations/${API.orgPlaceholder}/commerce/v1/products`;
 
-    get(productsRequest: New<ProductsRequestModel>) {
-        return this.api.post<ProductsResultModel>(`${Products.baseUrl}/listing`, productsRequest);
+    get(productsRequest: New<ProductsRequestModel>, refreshCache: boolean = false) {
```

Comment:
Detail but you don't really need to specify it's a boolean when it's equal to false.

Reply:
```suggestion
    get(productsRequest: New<ProductsRequestModel>, refreshCache = false) {
```